lib/beaker/dsl/install_utils/module_utils.rb

```diff
@@ -133,8 +133,10 @@ def copy_module_to(one_or_more_hosts, opts = {})
               cur_path = File.join(target_module_dir, source_name)
               if (cur_path != target_path)
                 if host.is_powershell?
+                  on host, "rmdir /S /Q #{target_path}"
                   on host, "move /y #{cur_path} #{target_path}"
                 else
+                  on host, "rm -Rf #{target_path}"
                   on host, "mv #{cur_path} #{target_path}"
```

Comment:
This should also be put into the host abstraction so that host.mv(cur_path, target_path) works.  Then this code will not need an OS-dependent if statement and will be easier to read/maintain.